main.go

```diff
@@ -7,11 +7,13 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"os/exec"
 )
 
 const (
-	path = "/payload"
-	port = ":3069"
+	path     = "/payload"
+	port     = ":3069"
+	tar_name = "test"
```

Comment:
Generate this from somewhere instead... Also it's clumsy to have function parameters overload global variable names

Reply:
Are you suggesting a config file or something like that?

Reply:
No. I think you should generate a somewhat random name. Getting a bunch of tars with the same image name may mess stuff up

Reply:
That makes sense. I'll look into using a combination of the username and reponame.